### DISCUSSION TITLE
Sonarr: only monitor selected season

### DISCUSSION THREAD

**Discussion by Ectalite** [sct/overseerr#3971] [Q&A]
Hi,
I saw that when a TV serie was requested through Overseerr, it would automaticaly monitor the whole tv serie in sonarr and such automaticaly monitor new seasons which I don't want.

Maybe I overlooked the option in sonarr, but I couldn't find a way that would only request the season I want to download without automaticaly monitor new seasons.

**Comment by piliener** [comment]
Under general settings in Overseerr there should be an option at the bottom to Allow partial series requests.
